src/app/spotlightPage/[spotlightId]/[displayId]/page.tsx

```diff
@@ -0,0 +1,86 @@
+'use client';
+
+import Link from 'next/link';
+import React, { useEffect, useState } from 'react';
+
+import { DisplayRow } from '../../../../types/types';
+import NavBar from '../../../../components/userComponents/navBar/navBar';
+import { fetchDisplayFromId } from '../../../../supabase/displays/queries';
+import { fetchDisplayfromSpotlight } from '../../../../supabase/tour_displays/queries';
+
+/**
+ *
+ * 
+ * @param root0
+ * @param root0.params
+ * @param root0.params.displayId
+ * @param root0.params.spotlightId
+ * @returns display page
+ */
+export default function Page({ params }: { params: { displayId: string, spotlightId: string } }) {
+  const [display, setDisplay] = useState<DisplayRow>({
+    coordinates: {0:0},
+    created_at: "N/A",
+    description: "N/A",
+    id: "0",
+    title: "N/A",
+    updated_at: "N/A",
+  });
+
+  const [otherDisplays, setOtherDisplays] = useState<DisplayRow[]>([]);
+
+  useEffect(() => {
+    /**
+     * @returns a display 
+     */
```

Comment:
for fetch data, add a more descriptive comment on what is happening within the function (2-3 sentences on average)